Patches/Rimeffect Core/ME_Machine.xml

```diff
@@ -0,0 +1,157 @@
+<Patch>
+
+	<Operation Class="PatchOperationFindMod">
+    <mods>
+		<li>Rim-Effect: Core</li>
+    </mods>
+		<match Class="PatchOperationSequence">
+		<operations>		
+
+		<li Class="PatchOperationAdd">
+		<xpath>Defs</xpath>
+		<value>
+
+  <ThingDef ParentName="BaseGunWithQuality">
```

Comment:
Why add these separately, if I may ask?

Reply:
Which line are you talking about? If its the Loki pistol, it is a infinite ammo workaround for player built robots/mechs being unable to pick up any ammo.